### DISCUSSION TITLE
Is there a way to convert text into a whispering voice?

### DISCUSSION THREAD

**Discussion by dexterhuh** [coqui-ai/TTS#3706] [General Q&A] [answered]
Is there a way to convert text into a whispering voice with a sample normal voice? Sorry, I'm a beginner. If there's a method, please give me some simple steps. Thank you very much!

**Comment by ghost** [comment, marked as answer]
Not with XTTS-zeroshot cloning, you'd need to finetune the model. But maybe 'Bark' can do it.

**Reply by dexterhuh** [reply]
Thank you very much. I will explore Bark. Is there any beginning tutorial for TTS finetuning the model?

**Reply by ghost** [reply]
I had a lot of problems getting to run the CoquiTTS framework to run with Cuda for simple inference in the first place, and that was installing via pip, not cloning the github repo, but according to their doc, you need to do exactly that, cloning the github repo first, when you want to do more than inference.

https://docs.coqui.ai/en/latest/finetuning.html
Video about finetuning from the Coqui team: https://www.youtube.com/watch?v=8tpDiiouGxc&pp=ygUPeHR0cyBmaW5ldHVuaW5n
There might also be more relevant information on the Thorsten-Voice channel from the legend himself:
https://www.youtube.com/playlist?list=PL19C7uchWZerUT0qIiEv7m2zXBs5kYl1L

**Reply by dexterhuh** [reply]
I have got CoquiTTS running smoothly. I will look at finetuning part! Thank you!

**Reply by pitxels** [reply]
Were you able to get the whispering voice with any model?
